Makefile

```diff
@@ -231,6 +231,9 @@ localintegration: varlink_generate test-binaries ginkgo
 remoteintegration: varlink_generate test-binaries ginkgo-remote
 
 localsystem:
+	# Wipe any trace of previous configuration; e.g. integration tests
+	# may have left an unusable storage driver or other state.
```

Comment:
Running in a F30 VM using the hack script now so this can be prooven.  There's no `~/.local` to start with (as a user), running integration tests now...

Reply:
...integration testing hit a failure in `podman exec with user only in container`.  Ignoring that, the only thing I see hanging around is the database:

```
[cevich@cevich-fedora-30-libpod-5744029755506688 libpod]$ ls -la ~/.local/share/containers/cache/
total 168
drwx------. 2 cevich cevich   4096 Jul  9 14:53 .
drwx------. 3 cevich cevich   4096 Jul  9 14:53 ..
-rw-------. 1 cevich cevich 262144 Jul  9 15:01 blob-info-cache-v1.boltdb
```

Trying the system-tests anyway to see if problem reproduces (without the fix in this PR)...

Reply:
okay, yep, that's it.

@edsantiago can you update the comment with something like:

```
	# Wipe previous user configurations and cache-database.
        # A user-selected storage driver has been observed to interfere
        # with system-testing.
```

My concern is that "may have" leaves this (and untold other problems) up to the readers imagination.  IMHO it's better to just be unambiguous about a (now) known/reproduced, 
specific problem.